### DISCUSSION TITLE
How to customize markdown

### DISCUSSION THREAD

**Discussion by JoshClose** [statiqdev/Discussions#48] [Q&A] [answered]
I'm using Statiq.Web. Is there a way to customize what markdown outputs? Each type of element I need to add classes to.

**Comment by daveaglick** [comment, marked as answer]
Absolutely - Statiq uses the amazing [Markdig](https://github.com/xoofx/markdig) renderer which includes support for a bunch of extensions. By default the "advanced" set of extensions is used, which includes [generic attributes](https://github.com/xoofx/markdig/blob/master/src/Markdig.Tests/Specs/GenericAttributesSpecs.md).

More specifically, to add a class to a paragraph you can do something like:

```
...

{.my-class}
This is a paragraph with the `.my-class` class on the `<p>` element.

...
```
